netlistsvg/build.sh

```diff
@@ -0,0 +1,22 @@
+
+#!/bin/bash
+
+set -e
+set -x
+
+if [ x"$TRAVIS" = xtrue ]; then
+	CPU_COUNT=2
+fi
```

Comment:
I'd drop that.  We should have a solution for proper multicore handling, done in a generic way

Reply:
Dropped